### ISSUE TITLE
v5.x support

### ISSUE THREAD

**Issue by askids** [appbaseio/gem#13]
hi,

I am new to ES. So looking at simpler ways to create/maintain the mappings. GEM looks a good option to try out. But now that v5.x no longer supports site plugins, is there a way to run this as a standalone site just like elasticsearch-head? I am using ES v5.x on windows. So please let me know how can I use GEM for my setup.

Thanks

**Comment by siddharthlatest** [comment]
@askids If you are just using it on a dev environment, there is a chrome extension that can be used -
 https://chrome.google.com/webstore/detail/gem/enmjddbghmojhlldblbblolfljndkkjn.

We don't have a docker image yet but it should be trivial to build one. I will look into this option.

**Comment by askids** [comment]
Thanks for the quick response. Unfortunately, our company blocks unauthorized plugins from being installed on Chrome due to security issues. If not, I would have tried that as the first option.

**Comment by naymore** [comment]
Hello there.
For me it's not working either. Neither the chrome plugin (thank you for that btw) nor building it from source. I can connect and there is a green bar on top of the application indicating some connection attempt but it never connects. My connection string is "http://localhost" with a locally running ES 5.4 (on windows for development purposes)

**Comment by siddharthlatest** [comment]
@naymore Did you change your ES configuration to allow the plugin access? You will need to do that in both cases - chrome extension and running locally.

**Comment by naymore** [comment]
@siddharthlatest, what exactly do you mean?
How do I enable plugin access? Is this even a plugin? Is there anything I have to put into my plugins directory? I'm confused.

**Comment by naymore** [comment]
Right... I thought installing the plugin was optional because it was one of the three "or" alternatives.
Where do I get the plugin from? Can't see any zips around...

**Comment by siddharthlatest** [comment]
@naymore I am referencing the chrome extension https://github.com/appbaseio/gem#get-the-chrome-extension. 

In your elasticsearch.yml file that you use to configure Elasticsearch, you need to add access for the chrome extension.

```
http.port: 9200
http.cors.allow-origin: "/.*/"
http.cors.enabled: true
http.cors.allow-headers: Authorization, X-Requested-With, Content-Type, Content-Length
http.cors.allow-credentials: true
```

**Comment by svanschalkwyk** [comment]
I can't seem to connect to http://localhost (port 9200)? 
Running the dev branch.

**Comment by neverbestboy** [comment]
GEM not connect with ES 5.4.1
